pubspec.yaml

```diff
@@ -57,9 +57,9 @@ flutter:
   uses-material-design: true
 
   # To add assets to your application, add an assets section, like this:
-  # assets:
-  #   - images/a_dot_burr.jpeg
-  #   - images/a_dot_ham.jpeg
+  assets:
```

Comment:
As mentioned above, please remove the images again in favor of the flutter material icons.

Reply:
done